### PR TITLE
Docs: corrected data source management information

### PR DESCRIPTION
**Pull request by usmangt**
Making minor changes after the PR merged on Data sources and Data source administration.

https://github.com/grafana/grafana/pull/83712

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/_index.md [hunk 1]
```diff
@@ -30,8 +30,14 @@ Panels can be dragged, dropped, and resized to rearrange them on the dashboard.
...
 
-- For more information about adding and managing data sources as an administrator, refer to [Data source management][].
-- For details about using specific data sources, refer to [Data sources][].
+- For details about using data sources, refer to [Data sources][].
+
+- For more information about managing data sources as an administrator, refer to [Data source management][].
+  
+  {{% admonition type="note" %}}
+  Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud(/docs/grafana-cloud).
+  {{% /admonition %}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/_index.md:39]: You can have use and manage data sources in Grafana OSS, Enterprise, or Cloud. What is this note referring to specifically?

**Review by imatwawana — CHANGES_REQUESTED** [review]

File: docs/sources/panels-visualizations/_index.md [hunk 2]
```diff
@@ -30,8 +30,14 @@ Panels can be dragged, dropped, and resized to rearrange them on the dashboard.
...
 Before you add a panel, ensure that you have configured a data source.
 
-- For more information about adding and managing data sources as an administrator, refer to [Data source management][].
-- For details about using specific data sources, refer to [Data sources][].
+- For details about using data sources, refer to [Data sources][].
+
+- For more information about managing data sources as an administrator, refer to [Data source management][].
+  
+  {{% admonition type="note" %}}
+  Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud(/docs/grafana-cloud).
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/_index.md:38]: Fixing broken link
```suggestion
  Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud](/docs/grafana-cloud).
```

File: docs/sources/panels-visualizations/_index.md (hunk 1 above)
Comment (usmangt) [reply docs/sources/panels-visualizations/_index.md:39]: Actually Managing Data Sources feature is only available in Grafana Enterprise and Cloud as there you can manage special settings, permissions, roles etc which is not available in OSS

**Review by imatwawana — COMMENTED** [review]
I've left a suggestion for rewording the note to make it clearer.

**Review by imatwawana — APPROVED** [review]

**Pull request merged**